.github/workflows/package.yml

```diff
@@ -0,0 +1,79 @@
+name: package
+
+on:
+  push:
+    tags:
+      - '**'
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - { os: 'debian', dist: 'stretch' }
+          - { os: 'debian', dist: 'buster' }
+          - { os: 'debian', dist: 'bullseye' }
+          - { os: 'el', dist: '7' }
+          - { os: 'el', dist: '8' }
+          - { os: 'fedora', dist: '30' }
+          - { os: 'fedora', dist: '31' }
+          - { os: 'fedora', dist: '32' }
+          - { os: 'fedora', dist: '33' }
+          - { os: 'fedora', dist: '34' }
+          - { os: 'opensuse-leap', dist: '15.1' }
+          - { os: 'opensuse-leap', dist: '15.2' }
+          - { os: 'ubuntu', dist: 'xenial' }
+          - { os: 'ubuntu', dist: 'bionic' }
+          - { os: 'ubuntu', dist: 'focal' }
+          - { os: 'ubuntu', dist: 'groovy' }
+          - { os: 'ubuntu', dist: 'hirsute' }
+
+    steps:
+      - name: Clone the module
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Clone the packpack tool
+        uses: actions/checkout@v2
+        with:
+          repository: packpack/packpack
+          path: packpack
+
+      - name: Create packages
+        env:
+          DEBIAN_FRONTEND: noninteractive
+          OS: ${{ matrix.platform.os }}
+          DIST: ${{ matrix.platform.dist }}
+        run: ./packpack/packpack
+
+      - name: Deploy packages
+        env:
+          OS: ${{ matrix.platform.os }}
+          DIST: ${{ matrix.platform.dist }}
+          RWS_URL: ${{ secrets.RWS_URL }}
+          RWS_USER: ${{ secrets.RWS_USER }}
+          RWS_PASSWORD: ${{ secrets.RWS_PASSWORD }}
+        run: |
+          case ${{ github.ref }} in
+            refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*)
+              REPO_S3_DIR=pre-release
+              ;;
+            refs/tags/*)
+              REPO_S3_DIR=release
+              ;;
+          esac
```

Comment:
We have no alpha/beta/rc in modules. So we deploy only to release/modules. Please, make it explicit in the code and ensure that this release/modules repository is what we recommend to our donwload instructions.

Reply:
Fixed